src/functions/index.js

```diff
@@ -0,0 +1,10 @@
+export const getSelectOptions = (records) => (option) => {
+	const options = records.flatMap((record) => {
+		if (record[option].length > 1) {
+			return record[option].split(' | ')
+		} else {
+			return record[option]
+		}
+	})
+	return [...new Set(options)]
+}
```

Comment:
What if instead of `option` you get an `options` array of strings `[genre, style, year, etc]`
then you do a forEach or reduce with this same logic you have over this array and at the end return and object like 
```
{ "genre": genreArray, "style": styleArray, ... }
```
This will avoid having to call the function several times for each option but still keeps it flexible to pass less/more options